draft-ietf-lamps-csr-attestation.md

```diff
@@ -494,37 +495,37 @@ EvidenceStatement ::= SEQUENCE {
    hint   UTF8String OPTIONAL
 }
 ~~~
+{: #sequence-fixme title="Figure Name FixMe3"}
 
-The type is on OID indicating the format of the data contained in stmt.
+In Figure {{sequence-fixme}}, type is an OID that indicates the format of the value of stmt.
```

Comment:
```suggestion
In {{sequence-fixme}}, type is an OID that indicates the format of the value of stmt.
```

Reply:
Thanks. Should have re-checked the rendering